packages/tenant-process/src/services/tenantService.ts

```diff
@@ -669,37 +758,19 @@ async function assignCertifiedAttribute({
 
 async function reevaluateTenantKind({
```

Comment:
I still found this name misleading.
Maybe I prefer something like `getTenantKindAndUpdateStatus`.
Any opinions or other suggestions?